core/app/models/mno_enterprise/tenant_config.rb

```diff
@@ -0,0 +1,366 @@
+# frozen_string_literal: true
+module MnoEnterprise
+  # Frontend configuration management
+  class TenantConfig
+    # == Constants ============================================================
+    # JSON schema for the configuration
+    # This is used to provide default values and to generate the form in the frontend
+    # This *MUST* be updated any time a new feature flag is added
+    CONFIG_JSON_SCHEMA = {
```

Comment:
We may want to load this from a file eventually....otherwise rubocop is going to complain on the number of lines in this class!